test/testinput/bufr_ncep_adpsfc_nc000007.yaml

```diff
@@ -0,0 +1,156 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+# #
+# # This software is licensed under the terms of the Apache Licence Version 2.0
+# # which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "./testinput/gdas.t12z.adpsfc.nc000007_thinned.tm00.bufr_d"
+
+      exports:
+        variables:
+          timestamp:
+            datetime:
+              year: "[*/YEAR[1], */YEAR[2]]"
+              month: "[*/MNTH[1], */MNTH[2]]"
+              day: "[*/DAYS[1], */DAYS[2]]"
+              hour: "[*/HOUR[1], */HOUR[2]]"
+              minute: "[*/MINU[1], */MINU[2]]"
```

Comment:
???? Only YEAR[1] will ever apply (picks the first one it finds in the BUFR subset tree heirarchy).